### DISCUSSION TITLE
Is it possible to show entire file path in the title bar?

### DISCUSSION THREAD

**Discussion by nickel8448** [ahrm/sioyek#1140] [Q&A] [answered]
Hello,

I use sioyek on Mac. I would like to somehow show the entire file path in the title bar. By any chance is that possible?

Thanks

**Comment by ahrm** [comment]
Currently it is not possible, it will be added in the next version.

**Comment by nickel8448** [comment, marked as answer]
Thanks, @ahrm 

I went through the documentation again (great job on writing such detailed documentation) and found option `show_doc_path`. Showing the full path in the recent window is fine by me. I can search for the current file name and it shows the path.

While not the most efficient process but it does work.
